lua/colorbox/controller.lua

```diff
@@ -138,6 +140,26 @@ M._parse_args = function(args)
   return opts
 end
 
+M.shuffle = function()
+  local ColorNamesList = runtime.colornames()
+  if #ColorNamesList > 0 then
+    local logger = logging.get("colorbox")
+    local previous_track = track.previous_track()
+    local i = previous_track ~= nil and previous_track.color_number or 0
+    local color = track.get_next_color_name_by_idx(i)
+    logger:debug(
+      string.format(
+        "|shuffle| color:%s, i:%d, ColorNamesList(%d):%s",
+        vim.inspect(color),
+        vim.inspect(i),
+        vim.inspect(#ColorNamesList),
+        vim.inspect(ColorNamesList)
+      )
+    )
+    loader.load(color)
```

Comment:
This could be:
```lua
M.shuffle = function()
      builtin_policy.shuffle()
end
```

Reply:
> This could be (?)
> 
> ```lua
> M.shuffle = builtin_policy.shuffle
> ```

Correct, but the `builtin_policy.shuffle` may not be directly invoked as an API.

Reply:
Also please add `shuffle` sub command to the doc - usage section: https://github.com/linrongbin16/colorbox.nvim?tab=readme-ov-file#-usage